getting_started/first_3d_game/08.score_and_replay.rst

```diff
@@ -164,7 +164,7 @@ There, we increment the score and update the displayed text.
 
 The second line uses the value of the ``score`` variable to replace the
 placeholder ``%s``. When using this feature, Godot automatically converts values
-to text, which is convenient to output text in labels or using the ``print()``
+to text, which is convenient to output text in labels or to use the ``print()``
```

Comment:
```suggestion
to text, which is convenient when outputting text in labels or the ``print()``
```

Reply:
I agree that my initial correction doesn't work grammatically. I think your suggested change doesn't work either. What about this:
```suggestion
to text, which is convenient when outputting text in labels or when using the ``print()``
```

Reply:
Yeah, that should be fine.

Reply:
Does this mean this can now be merged?

Reply:
No, the suggested changes weren't made, and by now this PR also needs to be rebased.